applications/sofa/gui/qt/viewer/qgl/QtGLViewer.cpp

```diff
@@ -138,8 +138,8 @@ QtGLViewer::QtGLViewer(QWidget* parent, const char* name, const unsigned int nbM
 
     setManipulatedFrame( new qglviewer::ManipulatedFrame() );
     //near and far plane are better placed
-    camera()->setZNearCoefficient(0.001);
-    camera()->setZClippingCoefficient(5);
+    camera()->setZNearCoefficient(0.005);
+    camera()->setZClippingCoefficient(2);
```

Comment:
better parameters?

Reply:
I just set the default parameters from QGLViewer documentation. (and the previous ones was messing up my scene :p )